gtest_parallel.py

```diff
@@ -559,7 +559,16 @@ def main():
   if options.dump_json_test_results is not None:
     test_results = CollectTestResults(options.dump_json_test_results)
 
-  save_file = os.path.join(os.path.expanduser("~"), ".gtest-parallel-times")
+  if sys.platform == 'win32':
+    default_cache_path = os.path.join(os.path.expanduser('~'), 'AppData', 'Local')
+    cache_path = os.environ.get('LOCALAPPDATA', default_cache_path)
+  else:
+    # We don't use xdg module since it's not a standard.
+    default_cache_path = os.path.join(os.path.expanduser('~'), '.cache')
+    cache_path = os.environ.get('XDG_CACHE_HOME', default_cache_path)
+
+  save_file = os.path.join(cache_path, 'gtest-parallel')
```

Comment:
After some more thinking... Since you don't want to exhibit this cache to end user at all then XDG_CACHE_HOME should not be used for managing "gtest-parallel" cache. If XDG_CACHE_HOME is changed for any reason we obey it but not more than that. We should not assume that it's changed for gtest_parallel.py exclusively and create this directory if it doesn't exist. Another reason is that changing of LOCALAPPDATA on Windows is not that easy and normally applications don't change it. So, probably we should not create LOCALAPPDATA if it doesn't exist too.